samples/bluetooth/fast_pair/locator_tag/README.rst

```diff
@@ -340,6 +340,28 @@ See :ref:`ug_bt_fast_pair_provisioning` for details.
    You cannot use the sample without the Fast Pair provisioning data.
    Programming the device with the sample firmware without providing the proper Fast Pair provisioning data results in assertion failure during boot.
 
+Release build
+=============
+
+You can build the sample in a release variant using the ``-DFILE_SUFFIX=release`` flag in your build command to learn about the configuration changes that you should apply to prepare the production build of your end product.
```

Comment:
```suggestion
You can build the sample in a release variant setting the ``FILE_SUFFIX=release`` CMake option.
That would result in using :file:`prj_release.conf` configuration file instead of :file:`prj.conf` configuration file for the build.
Check content of both files to learn about the configuration changes that you should apply to prepare the production build of your end product.
```
I would inform that user should check file content for details here.

Reply:
Also we should link to doc section about setting CMake options here (keep in mind that many customers do not build from CLI)